views/templates/admin/linkwidget/helpers/form/form.tpl

```diff
@@ -295,6 +265,41 @@
               return false;
             });
           });
+          /** get rankLink list */
+          function rankList()
+          {
+            var list = [];
+            $('[name=links] >td >input[type=number]').each(function ()
+            {
+              if ("" !== $(this).val())
+                list.push(parseInt($(this).val()));
+            });
+
+            return list;
+          }
+
+          /** verify rank of link */
+          function checkRank(element)
+          {
+            var rank = parseInt(element.value);
+            if (0 <= rank) {
+              if ($.inArray(rank, listLink) != -1) {
+                  element.value = "";
+              }
+            } else {
+              element.value = "";
+            }
+            listLink = rankList();
+          }
+
+          /** check the checkedBox */
+          function checkedBox(formId, parent)
+          {
+            $('#' + formId + ' input[type=checkbox]').each(function ()
+            {
+              $(this).prop('checked', parent);
+            });
+          }
```

Comment:
This block of javascript needs to be updated to conform with our coding standards